ReactiveCocoaTests/Swift/SignalSpec.swift

```diff
@@ -1484,5 +1484,263 @@ class SignalSpec: QuickSpec {
 				expect(error).to(equal(TestError.Default))
 			}
 		}
+		
+		describe("combinePrevious") {
+			
```

Comment:
Can you please match the style of the rest of this file, and remove the blank lines here and at the end of the `describe`?